exe/grnlip

```diff
@@ -11,7 +11,7 @@ end
 groonga_exe = ARGV.shift
 db_path = ARGV.shift
 basename = File.basename(db_path)
-db_encoding = ARGV.shift
+db_encoding = ARGV.shift.downcase
```

Comment:
I re-considered that this makes unclear command line options.
Because `uTf-8` and `utF-8` are also accepted here.
We should consider db_encoding option specification. :confounded: